difflint/lib.py

```diff
@@ -2,24 +2,44 @@
 from datetime import datetime
 from difflib import unified_diff
 from io import StringIO
-from os import unlink
+from os import getcwd, unlink
+import os.path
```

Comment:
I've been thinking about refactoring the imports.  I hate the difference here.  I like Philip's approach and will likely do that in a commit that I'll append to this PR.